js/foundation.util.touch.js

```diff
@@ -140,15 +140,10 @@ class Touch {}
   };
 
   Touch.init = function($) {
```

Comment:
Yeah I agree - I think there's a lot of thinking we need to do about dependencies.  One of the targets with 7.0 (and I'd like to get us there via incremental change & refactor through 6 series) is to make Foundation more "pluggable", having our dependencies (including jQuery) accessed more explicitly via interfaces and wrappers, which should allow us to swap them in and out more easily.  For example, if we have a dom manipulation interface with a jquery implementation, you could then also create an angular implementation, etc.  In essence separating out the logical "this is what should change in the UI" from the implementation "this is how we do it".  Maybe we could do something similar with touch handling